pkgs/development/tools/sshs/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "sshs";
+  version = "3.2.0";
+
+  src = fetchFromGitHub {
+    owner = "quantumsheep";
+    repo = pname;
+    rev = version;
+    sha256 = "D9doNVb2sTnzM8tF8cSJbIoaIYjGurkUHEyhcE3OqQg=";
+  };
+
+  vendorSha256 = "QWFz85bOrTnPGum5atccB5hKeATlZvDAt32by+DO/Fo=";
+
```

Please add ldflags as specified in the [Makefile](https://github.com/quantumsheep/sshs/blob/master/Makefile#L12).

```suggestion

ldflags = [ ]; # Flags should go here
```

---

resolved in 3fae311